atlasdb-client/build.gradle

```diff
@@ -72,7 +72,7 @@ dependencies {
   compile group: 'com.palantir.common', name: 'streams'
   compile group: 'com.github.rholder', name: 'guava-retrying'
   compile group: 'org.rocksdb', name: 'rocksdbjni'
-  compile group: 'com.palantir.goethe', name: 'goethe'
+  implementation group: 'com.palantir.goethe', name: 'goethe'
```

Comment:
We might as well move javapoet to `implementation` while we're here

Reply:
Done